### PR TITLE
Update documentation to include ad-hoc tasks

### PR DESCRIPTION
**Pull request by RouganStriker**
Refs #SDE-4417

TODO:
- [x] Set milestone label in pull request (SDE version)
- [x] Updated changelog.md
- [x] Completed documentation for the new endpoint.
- [x] Feature has been merged into SDE develop.

### CODE REVIEW

File: source/includes/tasks.md [hunk 1]
```diff
@@ -341,9 +346,7 @@ Content-Type: application/json
 This endpoint retrieves a single Task resource, as specified by the id parameter.
 
 **`GET /api/v2/projects/{project_id}/tasks/{task_id}/`**
-
-
-
+---
```
Comment (shawngustaw) [inline source/includes/tasks.md]: Is this intentional? Renders a bit weirdly.

**Comment by shawngustaw** [comment]
+1 for actually using valid JSON in examples

File: source/includes/tasks.md [hunk 2]
```diff
@@ -429,12 +433,78 @@ priority       | Yes      | The priority value from 0-10
...
 
+Parameter      | Description
+---------------|------------------
+project_id     | The id of the project the new task belongs to
 
+### Payload
 
+Fields          | Required | Description
+----------------|----------|---------------
+task_id         | Yes      | The id of the library task to add to the project.
```
Comment (RouganStriker) [inline source/includes/tasks.md]: TODO: Add the optional fields status, artifact_proxy, assigned_to

File: source/includes/tasks.md (hunk 1 above)
Comment (RouganStriker) [reply source/includes/tasks.md]: I noticed the other sections had this too so I added it to make the styling consistent

**Comment by RouganStriker** [comment]
Add MALT API documentation

**Pull request merged**